scripts/install_dev_tools

```diff
@@ -261,3 +265,27 @@ if should_cli_be_installed "promtool" "${cli_tools_arr[@]}" && \
       mv "${VENV}/bin/${DIR_NAME}/promtool" "${VENV}/bin/"
       rmdir "${VENV}/bin/${DIR_NAME}"
 fi
+
+# Conftest CLI
+if should_cli_be_installed "conftest" "${cli_tools_arr[@]}" && \
+    ! [ -x "$(command -v "${VENV}/bin/conftest")" ]; then
+      CONFTEST_CLIENT_URL=$(get_download_url_from_github_api "${CONFTEST_CLIENT_API_URL}") \
+                    || (echo "$CONFTEST_CLIENT_URL"; cleanup_and_exit 2)
+      download_file_from_url "${CONFTEST_CLIENT_URL}" "${DWN_DIR}"
+      CONFTEST_CLIENT_PATH="${DWN_DIR}"/$(basename "${CONFTEST_CLIENT_URL}")
+      extract_file_to_dir "${CONFTEST_CLIENT_PATH}" "${VENV}/bin/" "conftest"
+fi
+
+# Install bats via repository
+if should_cli_be_installed "bats" "${cli_tools_arr[@]}" && \
+    ! [ -x "$(command -v "${VENV}/bin/bats")" ]; then
+    if ! [ -x "$(command -v git)" ]; then
+        echo "git command not found"
+        exit 2
+    fi
+    git clone "${BATS_REPO_URL}" "${DWN_DIR}/bats"
+    pushd "${DWN_DIR}/bats"
+    ./install.sh "../../"
```

Comment:
This will be relative to the `DWN_DIR` from `mktemp`, which is system-dependent. This didn't work on macOS.

Should we pass the venv as the target?

The other thing is-- if this is a directory under `/tmp`, wouldn't this install it to the root fs? Shouldn't that fail unless this script is run as root?

Reply:
I am closing this comment as it is the same as previous one where the conversation happens.